packages/unison/unison.2.48.15v4/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "juergen@hoetzel.info"
+authors: [
+  "Jürgen Hötzel <juergen@hoetzel.info>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "mailto:unison-users@yahoogroups.com "
+# dev-repo: "https://github.com/bcpierce00/unison"
+depends: [ "ocaml" {<= "4.09"} ]
```

Comment:
```suggestion
depends: [ "ocaml" {< "4.10"} ]
```
Is this package really incompatible with possible fix versions of the compiler?

Reply:
Also if this is supposed to include `ocaml.4.09`, this is not the right constraint. The version `4.09` doesn't exist and is strictly lower than `4.09.0`